agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2279,6 +2303,13 @@ bool backhaul_manager::handle_multi_ap_policy_config_request(ieee1905_1::CmduMes
         ap_metrics_reporting_info.reporting_interval_s =
             metric_reporting_policy_tlv->metrics_reporting_interval_sec();
         ap_metrics_reporting_info.last_reporting_time_point = std::chrono::steady_clock::now();
```

Comment:
I think this is the equivalent to your static ap_metric_interval. If true, we should use one or the other, but not both. IMO, we should use member variables and not static variables.